src/main/java/core/basesyntax/dao/ShopDao.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface ShopDao {
+    Integer get(String fruit);
```

Comment:
Now I see that you don`t use the prepared list of fruits like in the last version of the program. So use there optional

Reply:
I suppose optional is redundant, I check wether the fruit is legal before the dao.get call